internal/app/upgradeagent/utils.go

```
@@ -35,3 +38,18 @@ func GetSolutionAndServiceName(containerName string) (string, string, error) {
 
 	return nameParts[0][1:], nameParts[1], err
 }
+
+// GenUUID generate UUID string
+func GenUUID() string {
+	u4, err := uuid.NewV4()
+	if err != nil {
+		log.Error(err)
```

In case of error, we should return the error.